### PR TITLE
Undo last point feature

### PR DESCRIPTION
**Pull request by bsekachev**
Good day, 
I added an ability to remove last point from poly shapes by svgDrawedInstance.draw('undo')
Could you review my patch?

### CODE REVIEW

File: src/lineable.js [hunk 1]
```diff
@@ -80,7 +80,7 @@
 
             this.set.clear();
 
-            for (var i = 0; i < array.length; ++i) {
+            for (var i = 0; i < array.length - 1; ++i) {
```
Comment (bsekachev) [inline src/lineable.js:83]: Last element is current position, right? We no need draw circle for it

Comment (Fuzzyma) [reply src/lineable.js:83]: Thats not entirely clear to me. Why dont we need a circle for the current position? Imo the current position should show the representation of what is there, when you click and add the current position to the pointlist. Hence the point at the current position

**Comment by Fuzzyma** [comment]
Very nice feature and clever code! I disagree with your opinion with the last point, though. I wrote my thoughts as comment :)

File: src/lineable.js (hunk 1 above)
Comment (bsekachev) [reply src/lineable.js:83]: Actually drawCircles() method called only from point() method, but not from calc() method. So two circles are being drown in last clicked point and no any circle which is being drown in last mouse position point (after mousemove). I provided screens. Is it bug? 
screenshot from 2018-07-12 15-12-41
screenshot from 2018-07-12 15-13-09

Comment (Fuzzyma) [reply src/lineable.js:83]: hm you are right. I guess we can merge it then :)

Comment (bsekachev) [reply src/lineable.js:83]: I could fix it in this patch, but it already merged. 
And I did't make minified version.
When will you do the next release?

Comment (Fuzzyma) [reply src/lineable.js:83]: uh thats what you mean. So there is a real bug in there. Jeah, it would be great if you could fix that one!

Comment (bsekachev) [reply src/lineable.js:83]: I will be work on it in the near future :)

Comment (Fuzzyma) [reply src/lineable.js:83]: cool, thank you!

**Pull request merged**